ConTabs/ConTabs.csproj

```diff
@@ -18,4 +18,7 @@
     <DebugType>full</DebugType>
     <DebugSymbols>True</DebugSymbols>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard1.3|AnyCPU'">
```

Comment:
Is this needed? Does it make ConTabs incompatible with projects targetting .NET Standard versions less than 1.3? I think we currently target 1.2...